build-tools/jest/setup.js

```diff
@@ -1,5 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-require('@testing-library/jest-dom/extend-expect');
-const { cleanup } = require('@testing-library/react');
-afterEach(cleanup);
+
+if (typeof window !== 'undefined') {
```

Comment:
Minor: would be nice to have a comment explaining why this condition is here

Reply:
done